src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, FigureDraw {
```

Comment:
The class name `Figure` is appropriate as it is a noun. Ensure that other classes and interfaces follow the same convention of not using verbs for names.